packages/serverpod/lib/src/server/log_manager/log_manager.dart

```diff
@@ -264,7 +264,7 @@ class SessionLogManager {
         return;
       }
 
-      assert(session.sessionLogs.currentEndpoint != null);
+      assert(session.endpoint != null);
```

Comment:
Quesiton: What happens if endpoint is null here? Should we throw or is that fine?

Reply:
probably, but I'm working on a new refactor making this assert redundant (removing the nullability of this variable).